### PR TITLE
Main script

### PR DESCRIPTION
**Pull request by emctackett**
`printPackageUpdates.rb`, when executed, displays version discrepancies on the command line.

### CODE REVIEW

File: packageUpdate.rb [hunk 1]
```diff
@@ -0,0 +1,17 @@
+require_relative 'getHomeBrewJson'
```
Comment (BaffourAdu) [inline packageUpdate.rb:1]: @emctackett Is this file needed to run the script?

Comment (emctackett) [reply packageUpdate.rb]: This file, `packageUpdate.rb` is actually a duplicate and I've removed it.

**Comment by emctackett** [comment]
@BaffourAdu it's not clear to me whether the files `bumpFormulae.rb`, `getHomeBrewJson.rb`, `getRepologyJson.rb`, and `compareHomeBrewAndRepology.rb` are still going to be needed and their functionality should be preserved.

This new script in particular, `printPackageUpdates.rb` merely displays data related to the out-of-date packages.  It doesn't use either of three aforementioned files.  Is the logic you are writing re pull-requests going to use the functionality in these files?

If no, I guess we should remove them?  If yes, I think this PR can be merged?

Thx! 🦄 🥳

**Comment by BaffourAdu** [comment]
@emctackett my functionality isn't going to use any of the 3 files. Should you see a reason for us to re-use you can create a helper file and add the necessary code or functionality there.

**Comment by emctackett** [comment]
@BaffourAdu Okay I'm going to remove them.  Just to clarify.. you aren't using `bumpFormulae.rb` either?

**Comment by emctackett** [comment]
I left `bumpFormulae.rb` for now. 

I've updated the portion of the `README` related to my script.  You will likely need to update the latter portion. 

We will also have to update the CI build, as its failing without the three deleted files.  I can try to do so when I get time.

**Comment by BaffourAdu** [comment]
@emctackett good call! With regard to the CI build failing, I will fix it and update it to this PR.

**Comment by BaffourAdu** [comment]
@emctackett I have fixed the CI and updated the Readme as well. Everything looks good now.
